pkg/serverless/invocationlifecycle/lifecycle_test.go

```diff
@@ -410,6 +587,65 @@ func TestTriggerTypesLifecycleEventForALB(t *testing.T) {
 	}, testProcessor.GetTags())
 }
 
+func TestTriggerTypesLifecycleEventForALB5xxResponse(t *testing.T) {
+	startDetails := &InvocationStartDetails{
```

Comment:
Is this test meaningfully different from TestTriggerTypesLifecycleEventForAPIGatewayWebsocket5xxResponse or TestTriggerTypesLifecycleEventForAPIGatewayNonProxy5xxResponse? The only difference I can see is using a different request payload, (but we are only testing the result based on the response). Perhaps we can refactor the tests to use a single common method, but swapping out the request payload, or delete some of the tests.

Reply:
No. Basically the same tests with slightly different request payload, but the responses are slightly different too. The refactored method might make the current logic a bit less straightforward imho.